### PR TITLE
Synchronize with latest version of google doc

### PR DESCRIPTION
**Pull request by kevinmpowell**
@kaelig I just did a line-by-line comparison with the latest Google Doc and what was currently in the repo. The Aliases and Groups sections were completely missing and several other sections were not up to date. I've made several changes here just to get aligned with the latest Google doc.

@jina I have not done any grammatical revisions (hemingwayapp or otherwise) yet, but those should be done on a future pass.

My aim with this PR is to be able to close out the google doc and fully migrate into this new git-based workflow without any data loss.

### CODE REVIEW

File: technical-reports/format/aliases.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "group name": {
+    "token name": {
+      "value": 1234
+    }
+  },
+  "alias name": {
+    "value": "{group name.token name}"
+  }
+}
+```
```
Comment (kaelig) [inline technical-reports/format/aliases.md:23]:
~~~suggestion
<aside class="example">

```json
{
  "group name": {
    "token name": {
      "value": 1234
    }
  },
  "alias name": {
    "value": "{group name.token name}"
  }
}
```

</aside>
~~~

File: technical-reports/format/groups.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      "token tres": {
+        "value": "token value 3"
+      },
+      "Token cuatro": {
+        "value": "token value 4"
+      }
+    }
+  }
+}
+```
```
Comment (kaelig) [inline technical-reports/format/groups.md:24]: Can you wrap this block into an example aside element? (see my other suggestion above)

File: technical-reports/format/groups.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+      "acid green": {
+        "value": "#00ff66"
+      },
+      "hot pink": {
+        "value": "#dd22cc"
+      }
+    }
+  }
+}
+```
```
Comment (kaelig) [inline technical-reports/format/groups.md:72]: Also can be wrapped to look like an example

File: technical-reports/format/groups.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      "primary": {
+        "value": "Comic Sans MS"
+      },
+      "secondary": {
+        "value": "Times New Roman"
+      }
+    }
+  }
+}
+```
```
Comment (kaelig) [inline technical-reports/format/groups.md:111]: Also can be wrapped to look like an example

File: technical-reports/format/groups.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+    "value": "#dd22cc"
+  },
+  "brand-typeface-primary": {
+    "value": "Comic Sans MS"
+  },
+  "brand-typeface-secondary": {
+    "value": "Times New Roman"
+  }
+}
+```
```
Comment (kaelig) [inline technical-reports/format/groups.md:130]: Also can be wrapped to look like an example

File: technical-reports/format/groups.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+Suggested ways tools may use this property are:
+
+- A style guide generator could render a section for each group and use the description as an introductory paragraph
+- A GUI tool that lets users browse or select tokens could display this info alongside the corresponding group or as a tooltip
+- Export tools could output this as a source code comment
+
+### TODO: List all supported properties at the group level
```
Comment (kaelig) [inline technical-reports/format/groups.md:80]: Is this meant to be a heading level 3?

File: technical-reports/format/groups.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+      "primary": {
+        "value": "Comic Sans MS"
+      },
+      "secondary": {
+        "value": "Times New Roman"
+      }
+    }
+  }
+}
+```
```
Comment (kaelig) [inline technical-reports/format/groups.md:165]: Also can be wrapped to look like an example

File: technical-reports/format/groups.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+
+```
+$brand-color-acid-green: #00ff66;
+$brand-color-hot-pink: #dd22cc;
+$brand-typeface-primary: "Comic Sans MS";
+$brand-typeface-secondary: "Times New Roman";
+```
+
+   </td>
+  </tr>
```
Comment (kaelig) [inline technical-reports/format/groups.md:180]: Also can be wrapped to look like an example (+ delete tr/td)

File: technical-reports/format/types.md [hunk 9]
```diff
@@ -1,12 +1,28 @@
...
+- Boolean
+- Null
+
+Additionally, this spec defines a number of more design-focused types. To set a token to one of these types, it MUST have a type property specifying the chosen type. Furthermore, that token’s value must then follow rules and syntax for the chosen type as defined by this spec.
+
+If the `type` property is absent, tools MUST treat values as one of the basic JSON types and not attempt to infer any other type from the value.
+
+If a `type` is set, but the value does not match the expected syntax then that token is invalid and an appropriate error should be displayed to the user. To put it another way, the type property is a declaration of what kind of values are permissible for the token. (This is similar to typing in programming languages like Java or TypeScript, where a value not compatible with the declared type causes a compilation error).
+
+## Design token types
```
Comment (kaelig) [inline technical-reports/format/types.md:19]: This heading may be redundant, considering the name of the section is "types"

**Review by kaelig — APPROVED** [review]
Awesome – Some minor formatting comments, which can be done later on. Feel free to merge as-is.

**Comment by jina** [comment]
> @jina I have not done any grammatical revisions (hemingwayapp or otherwise) yet, but those should be done on a future pass.

all good. I made a separate issue for this.

**Comment by kevinmpowell** [comment]
@kaelig I'll follow up on formatting issues later: https://github.com/design-tokens/community-group/issues/67

File: technical-reports/format/groups.md (hunk 6 above)
Comment (kevinmpowell) [reply technical-reports/format/groups.md:80]: Resolved by a recent merge.

**Pull request merged**